services/market/shared/market/priceUpdater.js

```diff
@@ -0,0 +1,121 @@
+/*
+ * LiskHQ/lisk-service
+ * Copyright © 2021 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ *
+ */
+const util = require('util');
+const BluebirdPromise = require('bluebird');
+const { Logger, CacheRedis } = require('lisk-service-framework');
+
+const config = require('../../config');
+
+const binance = require('./sources/binance');
+const bittrex = require('./sources/bittrex');
+const exchangeratesapi = require('./sources/exchangeratesapi');
+const kraken = require('./sources/kraken');
+
+const pricesCache = CacheRedis('market_prices', config.endpoints.redis);
+const logger = Logger();
+
+const targetPairs = config.market.targetPairs || [
+	'LSK_EUR', 'LSK_USD', 'LSK_CHF',
+	'BTC_EUR', 'BTC_USD', 'BTC_CHF',
+];
+
+const getRawPricesBySource = async () => {
```

Comment:
Given method can be simplified to:
```suggestion
const getRawPricesBySource = async () => ({
	binance: await binance.getFromCache(),
	bittrex: await bittrex.getFromCache(),
	exchangeratesapi: await exchangeratesapi.getFromCache(),
	kraken: await kraken.getFromCache(),
});
```